src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(numbers) {
```

Comment:
The parameter should be named to reflect that it is optional and could be 'undefined', such as 'number' with a default value of 'undefined'. This would make the function signature more descriptive.